### PR TITLE
Pipeline: CodeQL Fix

### PR DESCRIPTION
**Pull request by khkh-ms**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: eng/ci/official-build.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@ extends:
       name: 1es-pool-azfunc
       image: 1es-windows-2022
       os: windows
+    sdl:
+      codeql:
+         compiled:
+           enabled: true
+         runSourceLanguagesInSourceAnalysis: true
     stages:
```
Comment (arroyc) [inline eng/ci/official-build.yml:45]: do you need to add language and cadence as well as the variables?

**Review by arroyc — CHANGES_REQUESTED** [review]

**Comment by arroyc** [comment]
looks like a test is failing the build

File: eng/ci/official-build.yml (hunk 1 above)
Comment (khkh-ms) [reply eng/ci/official-build.yml:45]: No need to add the language.  Will leave default cadence to 72.

**Comment by khkh-ms** [comment]
All tests are passing.

**Review by arroyc — APPROVED** [review]

**Pull request merged**